_posts/2018-10-29-go.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Go over Nabla- App Safety meets Host Isolation
+---
+
+This blog presents how a Go application can be built and run as a Nabla container.
+We take an example of a web server application, part of the [nabla-demo-apps](https://github.com/Nabla-containers/nabla-demo-apps) repository.
```

Comment:
nit: nabla-containers instead of Nabla-containers. Just in case github becomes case sensitive in the future.